docs/enterprise-guide/saml-google.md

```diff
@@ -0,0 +1,18 @@
+# Setting up SAML with Google
```

Comment:
@flamber I took this from your notes here https://github.com/metabase/metabase/issues/21290#issuecomment-1080997005 and made some slight updates based on the Google docs. 

Let me know if it's correct and if there's anything else we should add!